notebooks/data_selection.ipynb

```diff
@@ -0,0 +1,1080 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    min_zenith = 0
</pre><p>astropy units</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cta-observatory/cta-lstchain/pull/934/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
Sorry for that, the units are present in the intermediate stage of the DL1 datacheck  (run-wise h5 files) but not on the final, night-wise  .h5. I still write that out with pandas.